src/resources/tls.cc

```diff
@@ -197,6 +197,10 @@ void BaseMbedTlsSocket::apply_certs(Process* process) {
   }
 }
 
+void BaseMbedTlsSocket::disable_certificate_verification() {
```

Comment:
validation - but I guess this is where the verification comes from :)

Reply:
done.